scout/web-app.md

```diff
@@ -0,0 +1,100 @@
+---
+description: The Docker Scout Web App helps review and share the analysis of images.
+keywords: scanning, analysis, vulnerabilities, Hub, supply chain, security, report, reports, web app
+title: Web App
```

Comment:
Have you discussed using a less generic product name for this? For example, Cockpit to keep with the nautical theme? ⛵ 

> A cockpit is a name for the location of controls of a vessel

Reply:
🌶️🌶️🌶️

This will definitely stir the pot 😄